python/semantic_kernel/connectors/ai/open_ai/services/open_ai_handler.py

```diff
@@ -21,18 +21,19 @@
 from semantic_kernel.connectors.ai.open_ai.services.open_ai_model_types import (
     OpenAIModelTypes,
 )
+from semantic_kernel.kernel_pydantic import KernelBaseModel
 
 logger: logging.Logger = logging.getLogger(__name__)
 
 
-class OpenAIHandler(ABC):
+class OpenAIHandler(KernelBaseModel, ABC):
     """Internal class for calls to OpenAI API's."""
 
     client: AsyncOpenAI
     ai_model_type: OpenAIModelTypes = OpenAIModelTypes.CHAT
-    prompt_tokens: int = Field(0, init=False)
-    completion_tokens: int = Field(0, init=False)
-    total_tokens: int = Field(0, init=False)
+    prompt_tokens: int = Field(0, json_schema_extra={'init': False})
```

Comment:
this shouldn't be necessary because it shouldn't be emitting a warning: https://docs.pydantic.dev/latest/api/fields/#pydantic.fields.Field

Reply:
Never mind, init is only valid for pydantic dataclasses, should be removed then, just `0` instead of `Field...`